src/cluster/cluster-registry.js

```diff
@@ -27,6 +27,7 @@ module.exports = class ClusterRegistry extends EventEmitter {
 	 */
 	constructor( options, connectionEndpoint ) {
 		super();
+		this.setMaxListeners( 20 );
```

Comment:
Interesting, it appears its because we have 11 distributed state registries. You can set the maxListeners to 11 then and we can add increase that each time to ensure nothing wierd is happening anywhere else